wmt_etl/tests/test_extract_loader.py

```diff
@@ -20,11 +20,12 @@ def cleanup_staging(connection):
 @pytest.mark.integration
 def test_should_import_extract():
     '''Should import valid dataframes to staging schema'''
+    engine = loader.get_db_engine()
+    connection = engine.connect()
+    cleanup_staging(connection)
```

Comment:
It looks like we need to be careful where we run these tests. If they are deleting records from staging tables and from `app.tasks`, we might have some data loss. Thoughts?

Reply:
Another question...What is the reason for the reordering?

Reply:
The issue is that tests were failing on first run but were successful on subsequent runs. The reason for this is
1) If we first run the `python start.py` command to create tables then that will populate table with 5 records.
2) when the test runs it adds the 3 records for its test and expects to have the three records back, instead it gets 8 records and it fails.
3) at the end of the test we clear the entire table anyway, removing both the test entries and the entries via start.py call. 

Removing the records prior to running the test has no impact as we will be clearing the table at the end of the test. This assumes that we don't run the start.py call and then run the test, which does make sense to do. 

I do think that we should only be removing the records that the test has entered and leave what ever was in the table prior to the test run.

Reply:
Yep, I'm ok with your addition to the code, my question is more around the implementation that was already there that I've only noticed now due to your update. If we run these on an environment where the worker has been running, will we lose all historical tasks in the `app.tasks` table when these tests run?